cirq-core/cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -30,7 +30,7 @@
 import numpy as np
 import scipy
 from matplotlib import pyplot as plt
-from cirq import circuits, devices, ops, protocols, sim, work
+from cirq import circuits, devices, ops, protocols, sim, work, value
```

Comment:
Alphabetical ordering.

```suggestion
from cirq import circuits, devices, ops, protocols, sim, value, work```